homeassistant/const.py

```diff
@@ -273,6 +273,7 @@ class Platform(StrEnum):
 # #### DEVICE CLASSES ####
 # DEVICE_CLASS_* below are deprecated as of 2021.12
 # use the SensorDeviceClass enum instead.
+DEVICE_CLASS_APPARENT_POWER: Final = "apparent_power"
```

Comment:
We don't need to add new ones here I think.

Reply:
We shouldn't, I added it because of the pattern in device_condition and tests

Reply:
The new constants are removed in a20726d